.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "4"
```

Comment:
Should I remove this version here? It's having different error than other versions.